router/src/server.rs

```diff
@@ -457,6 +512,16 @@ async fn generate(
         response.generated_text.generated_tokens as f64
     );
 
+    if std::env::var("REQUEST_LOGGER_URL").ok().is_some() {
```

Comment:
We grab this env var in a lot of place using a raw string. Can we move this to a single place where we grab it once during init and then just reuse a variable like `request_logger_url.is_some()` from then on?

Reply:
good point!